src/applications/ivc-champva/10-10D/components/Applicant/applicantFileUpload.js

```diff
@@ -1,76 +1,566 @@
+import React from 'react';
```

Comment:
This file now holds all the file upload UiSchemas along with their custom content